### ISSUE TITLE
Suggestion: AAC Codec

### ISSUE THREAD

**Issue by vpontin** [PC-Logix/OpenFM#46]
As reported in #45 , OpenFM seems to not have support of AAC streams.

I have virtually zero knowledge on Java, but searching on Google led me to this repository (https://github.com/DV8FromTheWorld/JAADec), which I understood as a open source AAC codec for  Java.

I dont know if is possible to integrate this to OpenFM, but given large webradio aggregators like ShoutCast uses MP3 or AAC in your index, i think this would be a great addition.

**Comment by CaitlynMainer** [comment]
Yeah I know DV8 rather well, I'll talk to him about this project.  Thanks for pointing it out.
